lib/bootloader_setup.pm

```diff
@@ -871,33 +871,35 @@ sub tianocore_http_boot {
 
 sub zkvm_add_disk {
     my ($svirt) = @_;
-    if (my $hdd = get_var('HDD_1')) {
-        my $basename = basename($hdd);
-        my $basedir  = svirt_host_basedir();
-        my $hdd_dir  = "$basedir/openqa/share/factory/hdd";
-        my $hdd_path = $svirt->get_cmd_output("find $hdd_dir -name $basename | head -n1 | tr -d '\n'");
-        die "Unable to find image $basename in $hdd_dir" unless $hdd_path;
-        diag("HDD path found: $hdd_path");
-        if (get_var('PATCHED_SYSTEM')) {
-            diag('in patched systems just load the patched image');
-            my $name        = $svirt->name;
-            my $patched_img = "$zkvm_img_path/$name" . "a.img";
-            $svirt->add_disk({file => $patched_img, dev_id => 'a'});
+    my $dev_id   = 'a';
+    
+    foreach my $n (1 .. get_var('NUMDISKS')) {
+        if (my $hdd = get_var('HDD_' . $n)) {
```

Comment:
Similar story here, I would suggest `get_required_var`.

Reply:
this is from bootloader_svirt - and this should be fine, since it's covered in the "if"
if there is no HDD_* var, it will not be executed

Reply:
Ah, true, we might want to have empty disks.

Reply:
Right, we might want empty disks. Not sure if we have any scenario where we attach one image and other disks are empty, but should work.